Source/Renderer/TextureMagnificationFilter.js

```diff
@@ -7,12 +7,38 @@ define([
     'use strict';
 
     /**
-     * @private
+     * Enumerates all possible filters used when magnifying WebGL textures, which takes places when zooming
+     * into imagery. Provides the possible values for the {@link ImageryLayer#magnificationFilter} property.
+     *
+     * @exports TextureMagnificationFilter
+     *
+     * @see TextureMinificationFilter
+     * @see ImageryLayer#magnificationFilter
```

Comment:
@lilleyse this `@see` - and the one in `TextureMinificationFilter.js` - is not needed and not customary.  Can you please remove in master?